mix.exs

```diff
@@ -41,13 +41,15 @@ defmodule Community.Mixfile do
   defp deps do
     [
       {:cowboy, "~> 1.0"},
+      {:ecto, "~> 2.0.0-rc", [override: true, optional: false, hex: :ecto, manager: :mix]},
       {:envy, "~> 0.0.2"},
-      {:ex_machina, "~> 0.6.1"},
+      {:ex_admin, github: "smpallen99/ex_admin"},
```

Comment:
So far so good, it saved me 2 hours of scaffolding but I haven't done anything outside of this PR so I'll let you know.